resources/scripts/worker.js

```diff
@@ -1,5 +1,3 @@
-importScripts('fuse.min.js');
```

Comment:
I don't think that the `new Fuse()` function in this file will work without import `fuse.min.js` file. We do need it.